Sources/swift-case-paths-benchmark/main.swift

```diff
@@ -26,6 +26,10 @@ let success = BenchmarkSuite(name: "Success") {
   $0.benchmark("Reflection") {
     precondition(reflection.extract(from: enumCase) == 42)
   }
+
+  $0.benchmark("Reflection (uncached)") {
+    precondition((/Enum.associatedValue).extract(from: enumCase) == 42)
```

Comment:
Felt like we should also have an understanding of the baseline when working with case paths that aren't cached in a larger system, _e.g._ a one-off `compactMap(/Result.success)`. It's still quite fast!

```
name                          time       std        iterations
--------------------------------------------------------------
Success.Manual                 32.000 ns ± 679.72 %    1000000
Success.Reflection            126.000 ns ± 391.25 %    1000000
Success.Reflection (uncached) 814.000 ns ± 131.98 %    1000000
Failure.Manual                 35.000 ns ± 767.22 %    1000000
Failure.Reflection             70.000 ns ± 519.43 %    1000000
Failure.Reflection (uncached) 552.000 ns ± 172.81 %    1000000
```